core/mono-samples/iOS/iOSSampleApp.csproj

```diff
@@ -42,7 +42,9 @@
       <Output TaskParameter="AppBundlePath" PropertyName="AppBundlePath" />
       <Output TaskParameter="XcodeProjectPath" PropertyName="XcodeProjectPath" />
     </AppleAppBuilderTask>
+  </Target>
 
+  <Target Name="LaunchApp" AfterTargets="BuildAppBundle" Condition="'$(RunOnCI)' == ''">
```

Comment:
nit: I'd rename this to `RunningOnCI` so it's not confused with "run the app during the CI build"